templates/local.settings.j2

```diff
@@ -1,2 +1,2 @@
 # {{ ansible_managed }}
-{{ cobbler_settings | to_nice_yaml(indent=2) }}
+{{ cobbler_settings }}
```

Comment:
I wonder if this would work?
```suggestion
{{ cobbler_settings | to_json(vault_to_text=True) | from_json | to_nice_yaml }}
```

Not sexy, but this would keep a nice output and won't trigger a configuration change for existing users.

Reply:
I tested and it works